### DISCUSSION TITLE
Add RetroBox as manager application

### DISCUSSION THREAD

**Discussion by xafero** [86Box/86Box#3446] [Approved Ideas]
**Is your feature request related to a problem? Please describe.**
I was searching for a manager application for the emulator, which works on Non-Windows.

**Describe the solution you'd like**
Please add a link to https://github.com/RetBox/RetroBox for users to see.

**Describe alternatives you've considered**
Maybe archive the old 86BoxManager project to avoid people writing new issues and problems there which will occur with new releases of the emulator, maybe when it switches its manager socket etc.

**Additional context**
None.

**Comment by kveroneau** [comment]
This is a bit funny, I was actually working on my own 86Box manager program in the middle of 2024 and came up with the same name as this guy did, "RetroBox", haha.  Guess I'll have to rename mine eventually, but I do say, I like the look and feel of this RetroBox over mine, which can be found here, if you were curious:  https://github.com/kveroneau/RetroBox

Although, since it did not work as well in practice as I wanted, I instead developed a more personal 86Box manager dedicated for Linux using it's amazing Btrfs file system support, which I did not call RetroBox, but instead called it "System Manager", and if you were curious about this one, it can be found here:  https://github.com/kveroneau/HobbyProjects/tree/main/SystemManager

System Manager is more for a 86Box bug tester, retro computing researcher, or someone who really wants to play around with a heck of a lot of retro 86Box systems.  It works amazingly well, and if you are a vet Linux user, who loves Btrfs, then it might not be a bad idea to check out, it compiles via FreePascal and Lazarus, which shouldn't be difficult.  It is currently heavily tailored to my PC, so you'll need to create your Btrfs and directory structure in a specific way or change the source itself to reflect your own system,
